ui/src/App.js

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  useEffect(() => {
+    (async () => {
+      const response = await fetch("http://localhost:5000/board");
```

Comment:
consider extracting this url as a constant.
i.e.
`host = http://localhost:5000`.

Reply:
se pone en el index.jsx o en el config... pero por ahora whatever

Reply:
@bcollazo esto debería ser App.jsx para representar que usa React JSX